config.default.yaml

```diff
@@ -29,7 +29,10 @@ countries: ["NG", "NE"]
   #["ZA"]  # South Africa
 
 # if True clusters to GADM shapes, if False Voronoi cells will be clustered
-alternative_clustering: False
+cluster_options:
+  alternative_clustering: False
+  distribute_cluster: ['load'] # ['load'],['pop'] or ['gdp']
```

Comment:
Why are you proposing a list notation? maybe we could use as codes simply load, pop or gdp, like:
distribute_cluster: load # load, pop or gdp

yaml interprets them as strings